_stbt/color.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+
+
+def get_color(region, frame=None):
+    """Get the average color of a region."""
+
+    import stbt
+
+    # Grab a frame if not provided by caller
+    frame = stbt.get_frame() if frame is None else frame
+
+    # Extract the requested region
+    frame = stbt.crop(frame.copy(), region)
```

Comment:
There's no need to copy the frame here as we don't modify it.

Reply:
P.S. `crop` returns a view into the original frame's data, it doesn't actually modify the frame.